content/docs/command-reference/remote/modify.md

```diff
@@ -582,6 +582,69 @@ more information.
 
 </details>
 
+<details>
+
+### Click for WebDAV
+
+- `token` - token for WebDAV server, can be empty in case of using
+  `user/password` authentication.
+
+  ```dvc
+  $ dvc remote modify --local myremote token mytoken
+  ```
+
+- `user` - username for WebDAV server, can be empty in case of using `token`
+  authentication. The order in which DVC searches for username is:
+
+  1. `user` specified in one of the DVC configs;
+  2. `user` specified in the url (e.g. `webdav://user@example.com/path`)
+
+  ```dvc
+  $ dvc remote modify --local myremote user myuser
+  ```
+
+- `password` - password for WebDAV server, can be empty in case of using `token`
+  authentication.
+
+  ```dvc
+  $ dvc remote modify --local myremote password mypassword
+  ```
+
+> The username, password, and token (may) contain sensitive user info.
+> Therefore, it's safer to add them with the `--local` option, so they're
+> written to a Git-ignored config file.
+
+- `ask_password` - ask each time for the password to use for `user/password`
+  authentication.
```

Comment:
```suggestion
- `ask_password` - ask each time for the password to use for `user`
  authentication.
```
Does this imply `password` is not needed and would have no effect when this is set?

Reply:
If `password` is set, it takes precedence. I think I have copied the explanation on this from `HTTP`? If not it should be added.

What is probably missing is the relation of this option to the `token` option: If `token` is set, `ask_password` does also have no effect. It should probably be mentioned in general that `user/password` and `token` are incompatible. I will add this soon.

Reply:
OK don't worry about HTTP, maybe it's incomplete. Please add any missing info for WebDAV's `ask_password` above, as you've explained.

Reply:
I updated this. Resolving